_pages/about-us/teams/products-and-platforms.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: Products and Platforms
 ---
-Products and Platforms is one of the 5 business units within 18F. We strive to incubate tools for digital government.
```

Comment:
Ah, gotcha — totally understand that you all are still working to define your mission. In the meantime, I recommend avoiding the term _incubate_, as it's jargony and could be offputting to some readers.